sshd-core/src/main/java/org/apache/sshd/certificate/OpenSshCertificateSignRequest.java

```diff
@@ -0,0 +1,136 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.sshd.certificate;
+
+import java.security.PublicKey;
+import java.time.Instant;
+import java.util.Collection;
+import java.util.List;
+
+import org.apache.sshd.common.config.keys.OpenSshCertificate;
+
+/**
+ * Holds all the data necessary to create a signed OpenSSH Certificate
+ */
```

Comment:
How about a builder (with fluent setters) for this instead of a signer and a sign request?
```
OpenSshCertificate cert =
  new OpenSshCertificateBuilder()
    .setType(OpenSshCertificate.Type.USER)
    .setPublicKey(keyToSign)
    .setValidAfter(Instant.now())
    .setValidBefore(null) // forever; don't expire
    .set....
    .sign(caKey);
```

Reply:
I like that idea, I will make that change

Reply:
This was implemented